### ISSUE TITLE
Add Submodule `Demo` into CQL repo

### ISSUE THREAD

**Issue by baseTwo** [FirelyTeam/firely-cql-sdk#448]
Before starting with this, pick up #428 first

Move demo projects into another repo and include them as submodule in the cql repo. See related #440 

See my notes on submodules: https://firely.atlassian.net/wiki/x/EAD5Yg

**Comment by baseTwo** [comment]
We already have 3 submodules, it's a burden already to juggle. Won't do until a better solution is found
